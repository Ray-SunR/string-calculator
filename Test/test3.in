n a abcabc\n			



la
wa
m b 0 a
lb
wb
		f c
i d c
lc
wc
ld
		wd
ea


     			eb
ec
     ed
q
