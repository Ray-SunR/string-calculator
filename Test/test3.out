8
abcabc\n
0

0

0

